generator/src/test/java/com/scottlogic/deg/generator/cucumber/InSet.feature

```diff
@@ -643,16 +643,17 @@ Scenario: Running a 'inSet' request alongside a non-contradicting matchingRegex
        | null   |
        | "test" |
 
-@ignore
-Scenario: Running a 'inSet' request alongside a contradicting matchingRegex constraint should fail with an error message
+Scenario: Running a 'inSet' request alongside a contradicting matchingRegex constraint should produce null
      Given there is a field foo
        And foo is in set:
        | "Test"  |
        | "Testt" |
        | "Tes7"  |
        And foo is matching regex /[a-z]{4}/
-     Then I am presented with an error message
-       And no data is created
+     Then the following data should be generated:
+       | foo  |
+       | null |
+
 
 Scenario: Running a 'inSet' request alongside a non-contradicting matchingRegex constraint should be successful
```

Comment:
This should say a non-contradicting containingRegex

Reply:
Renamed